docs/competition/arena/sr2017.md

```diff
@@ -0,0 +1,431 @@
+
+There are 16 A and 16 B tokens, requiring ~58m of red and ~58m of yellow tape respectively. There are 4 C tokens, requiring ~7.5m of white tape and ~7.5m of brown tape. All tape from [Gaffertape.com Industrial Grade/Cloth Tape](http://www.gaffatape.com/industrial-grade-heavy-duct-gaffer-cloth-tape/Default.aspx)
+
+### Carpet
+
+For the last few years we have used 'budget cord carpet' either in blue or red (depending on availability) from [More For Your Floor](http://www.moreforyourfloor.co.uk/blue-budget-cord-carpet.html). Previously we have ordered 2x 18x4m lengths and 1x 9x4m length, however we cut the 18m lengths in half. Since they charge per square metre, we can just order 5x 9x4m lengths for the same price and save our time cutting it up.
+
+### Arena walls
+
+The arena walls are 300mm tall made out of 9mm thick waterproof MDF in 1.144m sections. There are 7 sections per side of the arena, giving an internal size of 7.999m (7x1.144 - 0.009). Each section has two feet that stick out on the outside of the arena. These feet have water-filled 5l plastic jerry cans placed on them for ballast.
+
+The wall sections are held together with 3.5x25mm screws (2 screws per section for a total of 56 screws).
+
+We already own all of the wall sections, all of the screws and all of the jerry cans from previous years.
+
+### Network requirements
+
+There is a private LAN, the 'arena network', that connects all of the computers around the arenas, the shepherding area and the score entry area. This LAN requires points at:
+
+ * Each corner of each arena (8 off) requires two ethernet points that can reach up to 2m in the air from the floor.
+ * The shepherd area requires two ethernet points
+ * The score entry area requires two ethernet points (provide 4)
+ * The downstairs 'outside' display requires one ethernet point
+ * The upstairs 'outside' display requires one ethernet point. **This will be connected to the Newbury network, it will only have internet access and not access to the rest of the arena network.**
+
+This LAN connects to the Newbury network to get internet access. The connection to the Newbury network is via a NAT router. The NAT router also needs to provide DHCP leases with fixed hostnames (and provide DNS lookup of these hostnames) for the following MAC addresses:
+
+```
+b8:27:eb:89:e9:e0 pi-outside-0
+b8:27:eb:51:84:16 pi-shepherding-1
+b8:27:eb:e9:ec:98 pi-outside-2
+b8:27:eb:2e:84:e4 pi-arena-A0
+b8:27:eb:ec:0e:80 pi-arena-A1
+b8:27:eb:e4:7a:ef pi-arena-A2
+b8:27:eb:ed:f2:77 pi-arena-A3
+b8:27:eb:be:ce:73 pi-arena-B0
+b8:27:eb:b9:2b:29 pi-arena-B1
+b8:27:eb:3c:0b:c5 pi-arena-B2
+b8:27:eb:eb:9a:17 pi-arena-B3
+b8:27:eb:69:f2:04 pi-staging-11
+[awaiting from Peter] compbox-2017
```

Comment:
Not sure it's worth documenting this? Did we ever determine it, or should we just omit the line / block